### DISCUSSION TITLE
is this library maintained? Is there an alternative?

### DISCUSSION THREAD

**Discussion by praharshkp** [toluaina/pgsync#280] [Q&A] [answered]
Hi all! we have a site driven by Postgres. We also have an elastic search implementation to make filter/searches faster. We needed a framework to sync the indexes between these two. We tried pgsync 'https://github.com/ankane/pgsync/issues' and hit a roadblock, tried to get help online, failed, and realized that it's not actively supported anymore. Could you suggest a tool/framework for syncing between Postgres and elasticsearch?

**Comment by toluaina** [comment, marked as answer]
@praharshkp 

That library you posted above has nothing to do with this one.
It also has nothing to do with Elasticsearch.
The [README](https://github.com/toluaina/pgsync) described some alternatives
